README.md

```diff
@@ -32,6 +32,8 @@ docker run -ti --rm \
     -e APP_AUTHN_ENABLED=0 \
     -e APP_AUTHZ_ENABLED=0 \
     -e APP_STAT_ENABLED=0 \
+    -e APP_AGENT_LABEL=alpha \
```

Comment:
This one comes from the app's config (`agent_label`). In the examples by copying `App.toml.sample` into `App.toml`.

Reply:
My mistake, I've updated README for this example.